src/events/WindkraftForschung.ts

```diff
@@ -0,0 +1,33 @@
+import { defineEvent } from "../Factory"
+import { lawIsAccepted } from "../lawTools"
+import { markdown } from "../lib/utils"
+import { modify } from "../params"
+
+export default defineEvent({
+  title: "Durchbruch in der Windenergie-Forschung",
+  description: `Wissenschaftler haben eine neue Art der Energiegewinnung durch Wind entwickelt, mit der bestehende Windturbinen deutlich mehr Leistung bringen.`,
+
+  apply() {
+    return [modify("popularity").byValue(4), modify("electricityWind").byPercent(30)]
+  },
```

Comment:
30% within one leap is quite a bit. Maybe you could up `electricityWindOnShoreMaxNew` - but watch out of the laws setting that to new values...

Reply:
Maybe we need another base parameter for that which can be used as an multiplier for `electricityWindOnshoreMaxNew`?

Reply:
I create a new parameter `electricityWindEfficiency`

Reply:
Oh! I am sorry! You wrote "bestehende Windturbinen". Maybe it is fine like this. 30% per year might be a little high, but otherwise a yearly increase on top of the "Ausschreibungsverfahren" laws is probably correct, since the new technology needs to be implemented step by step.